great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -249,7 +253,8 @@ def get_batch_data_and_markers(
                 s3_object = s3_engine.get_object(Bucket=s3_url.bucket, Key=s3_url.key)
             except (ParamValidationError, ClientError) as error:
                 raise ge_exceptions.ExecutionEngineError(
-                    f"""PandasExecutionEngine encountered the following error while trying to read data from S3 Bucket: {error}"""
+                    f"""PandasExecutionEngine encountered the following error while trying to read data from S3 \
```

Comment:
I'd like to consider a longer line length to avoid this wrapping

Reply:
@donaldheppner But then we get `PyCharm` warnings.  Which way should we go?  Thanks.

Reply:
You can separate python strings in parens and they will automatically be joined.

![image](https://user-images.githubusercontent.com/13108583/196247598-51a244ae-431d-4a09-a705-209f9134775c.png)

Reply:
I think the warning will go away if you adopt our standard line length of 120 characters